R/rm.ridge.R

```diff
@@ -1,34 +1,56 @@
+#' @description
+#' Computes retention time intervals of a selected retention time range.
+#' @param rt full retention time vector.
+#' @param intensity Intensity vector.
+#' @param rt_range Selected retention time range vector.
+#' @return A list object:
+#' \itemize{
+#'   \item over_rt - upper retention time interval
+#'   \item under_rt - lower retention time interval
+#'   \item within_rt - intermediate retention time interval
```

Comment:
It seems to me that this description may be slightly misleading. As I understand it from the description, the function returns the values. However, `over_rt`, `under_rt`, and `within_rt` seem to be indices of the values in the `rt` vector.

Reply:
Indeed, they are indices. It has been updated with more accurate description ([0560ff6](https://github.com/RECETOX/recetox-aplcms/pull/171/commits/0560ff64c3690fad64f378709962e05684a01d19), [1e6afb2](https://github.com/RECETOX/recetox-aplcms/pull/171/commits/1e6afb2525c3fad5185b584a59e0c4adf5c62e1c)).